tests/functional/oauth_webchannel.js

```diff
@@ -39,13 +39,13 @@ define([
    * finish OAuth flows
    */
 
-  function testIsBrowserNotifiedOfLogin(context, shouldCloseTab, wantKeys) {
+  function testIsBrowserNotifiedOfLogin(context) {
     return FunctionalHelpers.testIsBrowserNotified(context, 'oauth_complete', function (data) {
       assert.ok(data.redirect);
       assert.ok(data.code);
       assert.ok(data.state);
-      assert.equal(data.closeWindow, shouldCloseTab);
-      if (wantKeys) {
+      assert.equal(data.closeWindow, context.shouldCloseTab);
+      if (context.wantKeys) {
```

Comment:
We should update this to have `wantsKeys`, instead of `wantKeys`. 
We use "wants" everywhere else https://github.com/mozilla/fxa-content-server/search?utf8=%E2%9C%93&q=wantsKeys